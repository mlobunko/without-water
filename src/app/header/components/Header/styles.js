import styled from 'styled-components';
import { NavbarBrand, NavLink } from 'reactstrap';
import imageSieve from '../../images/sieve.svg';

export const StyledNavbarBrand = styled(NavbarBrand)`
  background: url(${imageSieve}) no-repeat center center;
  background-size: contain;
  height: 2.5rem;
  width: 2.5rem;
`;

export const StyledNavLink = styled(NavLink)`
  &:hover {
    cursor: pointer;
  }
`;
