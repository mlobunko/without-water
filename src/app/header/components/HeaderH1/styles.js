import styled from 'styled-components';

export const StyledHeaderH1 = styled.h1`
  text-align: center;
  margin-top: 1rem;
  margin-bottom: 3rem;
`;

StyledHeaderH1.displayName = 'StyledHeaderH1';
