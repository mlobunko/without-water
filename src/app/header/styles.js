import styled from 'styled-components';
import { NavbarBrand, NavLink } from 'reactstrap';
import imageSieve from './images/sieve.svg';

export const StyledNavbarBrand = styled(NavbarBrand)`
  background: url(${imageSieve}) no-repeat center center;
  background-size: contain;
  height: 2.5rem;
  width: 2.5rem;
`;

export const StyledNavLink = styled(NavLink)`
  &:hover {
    cursor: pointer;
  }
`;

export const StyledHeaderH1 = styled.h1`
  text-align: center;
  margin-top: 1rem;
  margin-bottom: 3rem;
`;
