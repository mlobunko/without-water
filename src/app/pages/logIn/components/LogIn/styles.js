import styled from 'styled-components';

export const StyledButton = styled.div`
  text-align: center;
`;
StyledButton.displayName = 'StyledButton';

export const StyledForm = styled.form`
  margin: 0 auto;
  width: 90%;
  max-width: 400px;
`;

StyledForm.displayName = 'StyledForm';
