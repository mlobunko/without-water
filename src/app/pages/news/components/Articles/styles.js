import styled from 'styled-components';
import { Jumbotron } from 'reactstrap';

export const StyledArticlesSection = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  flex-grow: 1;
`;
StyledArticlesSection.displayName = 'StyledArticlesSection';

export const StyledCounterNews = styled.div`
  text-align: center;
`;
StyledCounterNews.displayName = 'StyledCounterNews';

export const StyledJumbotron = styled(Jumbotron)`
  max-width: 700px;
  && {
    background-color: rgb(250, 250, 250);
  }
`;
StyledJumbotron.displayName = 'StyledJumbotron';

export const StyledHeader = styled.h3`
  margin-bottom: 2rem;
`;
StyledHeader.displayName = 'StyledHeader';
