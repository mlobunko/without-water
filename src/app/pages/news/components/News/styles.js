import styled from 'styled-components';

export const StyledNewsComponent = styled.div`
  flex-grow: 1;
  display: flex;
  flex-direction: column;
`;
StyledNewsComponent.displayName = 'StyledNewsComponent';
