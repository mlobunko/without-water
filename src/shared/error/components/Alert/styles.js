import styled, { css } from 'styled-components';

export const AlertStyled = styled.div`
  ${props =>
    props.centerPage &&
    css`
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      height: 60vh;
    `};
`;
