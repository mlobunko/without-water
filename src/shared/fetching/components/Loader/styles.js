import styled, { css } from 'styled-components';
import imageLoader from '../../images/loader.svg';

export const StyledLoader = styled.div`
  ${props =>
    props.centerPage &&
    css`
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      height: 60vh;
    `};
`;

export const StyledImage = styled.div`
  background: url(${imageLoader}) no-repeat center center;
  background-size: contain;
  height: 3rem;
  width: 100%;
  margin-bottom: ${props => props.login && '1rem'};
`;
