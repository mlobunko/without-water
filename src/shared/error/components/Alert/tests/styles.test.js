import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { AlertStyled } from '../styles';

describe('AlertStyled component', () => {
  it('renders properly default', () => {
    const wrapper = shallow(<AlertStyled />);
    expect(wrapper).toMatchSnapshot();
  });
  it('renders properly with centerPage', () => {
    const wrapper = shallow(<AlertStyled centerPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
