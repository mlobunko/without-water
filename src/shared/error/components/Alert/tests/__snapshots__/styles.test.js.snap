// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertStyled component renders properly default 1`] = `
<div
  className=""
/>
`;

exports[`AlertStyled component renders properly with centerPage 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 60vh;
}

<div
  className="c0"
/>
`;
